src/server.js

```diff
@@ -1,7 +1,16 @@
 import express from "express"
+import cors from "cors"
+import { json, urlencoded } from "body-parser"
+import morgan from "morgan"
 
 const port = 3000
 const app = express()
+
+app.use(cors())
+app.use(json())
+app.use(urlencoded({ extended: true }))
+app.use(morgan("dev"))
```

Comment:
This is nice and succinct, I like the way you've managed to fit it all in the same line. Excellent work 😄 🎆 🕺 🎉 

I often prefer to be more explicit in my code, what would you think of the above?

Don't feel obliged to change but I won't be your bestest friend if you don't 😦 😢 😜 
```suggestion
const devLogger = morgan('dev');
app.use(devLogger)
```

Reply:
Thanks for the feedback and totally agree that being explicit is best practice! 👍 

This is a great idea, but perhaps it would be good to do this at a later stage when we have middleware refactored into its own file? 

As these are fairly standard modules familiar to most node developers, it might be premature at this stage to optimise. What do you think?

Still happy to do this if it meets the style guides for the organisation, still learning the ropes 😄

Reply:
Agree, I think the logic you put forth is sound (_ticks off christmas card list_)